lib/helper/QubitHelper.php

```diff
@@ -329,6 +329,9 @@ function render_b5_section_heading(
     $url = null,
     $linkOptions = []
 ) {
+    // Add extra span to allow inline styling
+    $text = render_b5_show_container('span', $text);
```

Comment:
@replaceafill ... if we didn't break the other theme section headings before, 45cbbcd will most likely do it. I tried to address it adding this extra span here, but let me know if something else is needed.